basecamp.go

```diff
@@ -193,6 +202,10 @@ func (c *Client) get(url string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	fmt.Println("BASECAMP REQ", url, "time", time.Since(start))
+	if 304 == resp.StatusCode {
```

Comment:
Can we use status code constants from `net/http`? I'm also in favour avoiding [Yoda conditions][1].

[1]: https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
Fixed